Add 1.21 Docs Shadows to GitHub teams

reylejano opened the pull request:
Add 1.21 Docs Shadows to `website-milestone-maintainers` in  sig-docs/teams.yaml and `release-team` in sig-release/teams.yaml GitHub teams

/assign @palnabarun @kikisdeliveryservice @savitharaghunathan @bai

/hold for feedback if shadows should be added to the `website-milestone-maintainers` team. 
I think they should have the ability to add the 1.21 milestone on kubernetes/website PRs.

savitharaghunathan commented on config/kubernetes/sig-release/teams.yaml:
```diff
@@ -43,6 +43,7 @@ teams:
     - cantbewong # VMware
     - caseydavenport # Network
     - cheftako # Cloud Provider
+    - chandanim123 # 1.21 Docs Shadow
```
```suggestion
    + ChandaniM123 # 1.21 Docs Shadow
```

https://github.com/kubernetes/org/issues/2436 will be closed today via https://github.com/kubernetes/org/pull/2441

savitharaghunathan replied:
Also, please swap   
   - cheftako # Cloud Provider
   - chandanim123 # 1.21 Docs Shadow

savitharaghunathan commented on config/kubernetes/sig-release/teams.yaml:
```diff
@@ -252,11 +256,11 @@ teams:
         - arunmk # 1.21 Enhancements Shadow
         - bai # 1.20 RT Lead Shadow
         - celestehorgan # 1.20 Release Notes Shadow
+        - chandanim123 # 1.21 Docs Shadow
```
```suggestion
    + ChandaniM123 # 1.21 Docs Shadow
```

reylejano replied:
Changes made and commits squashed

reylejano commented:
/retest

reylejano commented:
/retest

savitharaghunathan reviewed (commented):
/lgtm

I agree with shadows having permissions to add/remove milestones. 

@reylejano please make sure that the handbook reflects this and add a cleanup step for the same. Thank you.

reylejano commented:
/hold cancel

palnabarun reviewed (commented):
> if shadows should be added to the milestone maintainers team.

@reylejano -- I think access for adding/removing milestones on `k/website` is given through the `website-milestone-maintainers` group. Ref: https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml#L342

If the shadows don't need access to globally add/remove milestones, I think `website-milestone-maintainers` is the better place.

LMK what do you think.

/hold for resolution on the above

reylejano commented:
> > if shadows should be added to the milestone maintainers team.
> 
> @reylejano -- I think access for adding/removing milestones on `k/website` is given through the `website-milestone-maintainers` group. Ref: https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml#L342
> 
> If the shadows don't need access to globally add/remove milestones, I think `website-milestone-maintainers` is the better place.
> 
> LMK what do you think.
> 
> /hold for resolution on the above

@palnabarun I removed the shadows from the `milestone-maintners` and added them to `website-milestone-maintainers` in sig-docs/teams.yaml

palnabarun commented:
@reylejano -- Awesome! Thank you so much :)

/lgtm
/approve

palnabarun commented:
/hold cancel
